CONTRIBUTING.adoc

```diff
@@ -1,99 +1,105 @@
+IMPORTANT: There's a lot more to the Jenkins project than just code. For more information on the ways that you can contribute to the Jenkins project, see https://www.jenkins.io/participate/[Participate].
 
-## Getting started
+== Getting started
 
-1. Fork the repository on GitHub
-2. Clone the forked repository to your machine
-3. Install the necessary development tools. In order to develop Jenkins, you need the following:
-   - Java Development Kit (JDK) 11 or 17.
-     In the Jenkins project we usually use [Eclipse Temurin](https://adoptium.net/) or [OpenJDK](https://openjdk.java.net/), but you can use other JDKs as well.
-   - Apache Maven 3.8.1 or above. You can [download Maven here](https://maven.apache.org/download.cgi).
-     In the Jenkins project we usually use the most recent Maven release.
-   - Any IDE which supports importing Maven projects.
-   - Install [NodeJS 16.x](https://nodejs.org/en/). **Note:** only needed to work on the frontend assets found in the `war` module.
-     - Frontend tasks are run using [yarn](https://yarnpkg.com/). Run `npm install -g yarn` to install it.
-4. Set up your development environment as described in [Preparing for Plugin Development](https://www.jenkins.io/doc/developer/tutorial/prepare/)
+. Fork the repository on GitHub
+. Clone the forked repository to your machine
+. Install the necessary development tools. In order to develop Jenkins, you need the following:
+* Java Development Kit (JDK) 11 or 17. In the Jenkins project we usually use https://adoptium.net/[Eclipse Temurin] or https://openjdk.java.net/[OpenJDK], but you can use other JDKs as well.
+* Apache Maven 3.8.1 or above. You can https://maven.apache.org/download.cgi[download Maven here].
+  In the Jenkins project we usually use the most recent Maven release.
+* Any IDE which supports importing Maven projects.
+* Install https://nodejs.org/en/[NodeJS 16.x]. *Note:* only needed to work on the frontend assets found in the `war` module.
+** Frontend tasks are run using https://yarnpkg.com/[yarn]. Run `npm install -g yarn` to install it.
+. Set up your development environment as described in https://www.jenkins.io/doc/developer/tutorial/prepare/[Preparing for Plugin Development]
 
 If you want to contribute to Jenkins, or just learn about the project,
 you can start by fixing some easier issues.
 In the Jenkins issue tracker we mark such issues as `newbie-friendly`.
-You can find them by using this query (check the link) for [newbie friendly issues](<https://issues.jenkins.io/issues/?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20core%20AND%20labels%20in%20(newbie-friendly)>).
+You can find them by using this query (check the link) for https://issues.jenkins.io/issues/?jql=project%20%3D%20JENKINS%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20component%20%3D%20core%20AND%20labels%20in%20(newbie-friendly)[newbie friendly issues].
```

Comment:
Optional, a simpler URL for friendly issues and it presents the issues in a table with priority and plugin name.
```suggestion
You can find them by using this query (check the link) for https://issues.jenkins.io/secure/Dashboard.jspa?selectPageId=19342[newbie friendly issues].
```

Reply:
I've got some small other changes planned but for another PR.
I'd like to keep this one with the less change possible from the original, just converting the current content to Asciidoc.